src/pages/templates/pages/partial/dashboard_panel_popular_reads_guest_messages.html

```diff
@@ -0,0 +1,23 @@
+{% if data.view == 'assigned' %}
+    A list of recently assigned readings displays for students with a
+    Clusive account connected with a teacher.
+{% elif data.view == 'recent' %}
+    {% if data.all|length == 0 %}
+        You have no recent readings yet. Go to the 
+        <a href="{% url 'library_style_redirect' view='period' %}">library</a>
+        and get started in Clusive!
+    {% else %}
+        Revisit something you've read, or head to the
+        <a href="{% url 'library_style_redirect' view='period' %}">library</a>
+        for something new.
+    {% endif %}    
+{% elif data.view == 'popular' %}
+    {% if data.all|length == 0 %}
+       There are no readings popular with Clusive readers right now. Once they
+       do more reading, you will find popular titles here.
+    {% else %}
+        These readings are popular with other Clusive readers. See all popular
```

Comment:
We should drop the 'see all popular readings' link for now, since we don't have a library view for that yet.

Reply:
I was wondering about that since whenever it was clicked, the library page showed some list other than a list of popular books.  Okay, I'll drop it.